### ISSUE TITLE
no cuDNN stream anymore....

### ISSUE THREAD

**Issue by doc1977** [iperov/DeepFaceLab#5482]
## Expected behavior

To work normally as it did the last months

## Actual behavior

Every step in the DFL workflow that uses cuDNN produces same error message: 

F tensorflow/stream_executor/cuda/cuda_dnn.cc:194] Check failed: status == CUDNN_STATUS_SUCCESS (7 vs. 0)Failed to set cuDNN stream.

## Steps to reproduce

simply retry to run it :-P

## Other relevant information
Occured first after NVIDIA Driver update to most latest version. Tried a Rollback with deinstalling Drivers via DDU application and reinstalling old version but problem is persisting.

Windows 10
NVIDIA GTX 1660 Super
DeepFaceLab_NVIDIA_up_to_RTX2080Ti_build_11_20_2021

ACTUAL Driver/Version configuration:
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 511.79       Driver Version: 511.79       CUDA Version: 11.6     |
|-------------------------------+----------------------+----------------------+
| GPU  Name            TCC/WDDM | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  NVIDIA GeForce ... WDDM  | 00000000:01:00.0  On |                  N/A |
|  0%   51C    P8    20W / 125W |    506MiB /  6144MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+

**Comment by ghost** [comment]
Did you ever find the answer? If so, would you mind sharing it and closing this issue?

**Comment by doc1977** [comment]
No, never got nor found an answer, still doesn't work on my machine
